{"name": "test_getbyid_failure", "status": "passed", "start": 1729324713460, "stop": 1729324713460, "uuid": "abace308-8087-42e9-9a3f-92af1eaa1717", "historyId": "cbc29a25fe64c1c9081b3685cd78d352", "testCaseId": "cbc29a25fe64c1c9081b3685cd78d352", "fullName": "Exercise.exercise_test.TestExerciseServiceWithoutMock#test_getbyid_failure", "labels": [{"name": "parentSuite", "value": "Exercise"}, {"name": "suite", "value": "exercise_test"}, {"name": "subSuite", "value": "TestExerciseServiceWithoutMock"}, {"name": "host", "value": "DESKTOP-IDGJ8R7"}, {"name": "thread", "value": "2016-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Exercise.exercise_test"}]}